.github/workflows/CI.yml

```diff
@@ -148,10 +148,10 @@ jobs:
           - host: windows-latest
             target: x86_64-pc-windows-msvc
         node:
-          - '12'
```

Comment:
why did we want to drop support for Node 12 here?

Reply:
I've brought back Node 20 and updated the readme.